src/pages/Login/LoginForm.js

```diff
@@ -1,22 +1,56 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
 import './LoginForm.scss';
 
 class LoginForm extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+      isBtnActive: false,
+    };
+  }
+
+  handleInput = event => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  goMain = () => {
+    this.props.history.push('/main');
+  };
+
   render() {
+    const isLoginBtnActive =
+      this.state.id.includes('@') && this.state.pw.length > 4;
     return (
       <div className="loginForm">
         <form className="loginInput">
           <input
             className="loginId"
             type="text"
-            placeholder="아이디를 입력해주세요."
+            name="id"
+            placeholder="아이디 또는 이메일을 입력해주세요."
+            onChange={this.handleInput}
           />
           <input
             className="loginPw"
             type="password"
+            name="pw"
             placeholder="비밀번호를 입력해주세요."
+            onChange={this.handleInput}
           />
-          <button className="loginBtn" type="button">
+          <button
+            className={
+              isLoginBtnActive ? 'loginBtn loginBtnActive' : 'loginBtn'
+            }
```

Comment:
```suggestion
            className={
              'loginBtn' + isLoginBtnActive ? 'loginBtnActive' : ''
            }
```

```suggestion
            className={
              'loginBtn' + isLoginBtnActive || ''
            }
```

저는 이렇게를 말쓰드린 거긴 하지만 큰 리뷰는 아니라서 머지하겠습니다!

Reply:
앗 이런 말씀이셨군요!😅
피드백 감사합니다!🙂
리뷰 남겨 주신 부분 다시 한 번 참고하도록 하겠습니다!